packages/graph/hash_graph/Makefile.toml

```diff
@@ -63,3 +50,30 @@ yarn workspace @hashintel/hash-graph-client prettier --write .
 # ESlint enforces ordering, which enables us to rely on the output diff as a CI check.
 yarn workspace @hashintel/hash-graph-client fix:eslint
 """
+
+[tasks.build-docker]
+category = "Deploy"
+description = "Builds a docker image of the graph"
+command = "docker"
+cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}"
+args = ["build", "--tag", "graph", "--file", "../deployment/graph/Dockerfile", "--build-arg", "PROFILE=${CARGO_MAKE_CARGO_PROFILE}", "."]
+
+[tasks.docker-up]
+category = "Deploy"
+description = "Runs the docker image of the graph"
+cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}"
+script = """
+# We need the IP for the postgres deployment. This will be easier, when the graph docker image is started from docker-compose
+gateway=$(docker network inspect deployment_default | jq -r '.[0].IPAM.Config[0].Gateway')
+docker run --rm --name graph --env RUST_LOG=debug --network deployment_default --publish "4000:4000" --detach graph --host $gateway --rest-address 0.0.0.0:4000
+"""
+
+[tasks.docker-down]
+category = "Deploy"
+description = "Stops the docker image of the graph"
+command = "docker"
+cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}"
+args = ["kill", "--signal", "SIGKILL", "graph"]
```

Comment:
Since the readme shows using stop instead of kill now, do we want to change this to be stop too?

Reply:
Applied in 02ff56a59b2b90ba1878e0f6e4741f8b550fff55